webpack/__mocks__/foremanReact/components/common/Slot/index.js

```diff
@@ -0,0 +1 @@
+export default jest.fn;
```

Comment:
I think this has to be `() => () => {}` or maybe `()  => jest.fn`. You also don't have to mock Foreman components, have you tried not mocking it?

Reply:
> Yes, you can use the code from Foreman, all Foreman code is available during testing. You can of course still mock too.

thanks, I removed those mocks as the issue wasn't there actually